spec/factories/user_sets.rb

```diff
@@ -1,7 +1,7 @@
 module SupplejackApi
   FactoryBot.define do
     factory :user_set, class: SupplejackApi::UserSet do
-      association :record, factory: :record
+      association :record, factory: :record_with_fragment
```

Comment:
Did you have failing tests?  I'd say this file is one of the most changed in the test suite.